projects/common-form-elements/src/lib/multiple-dropdown/multiple-dropdown.component.ts

```diff
@@ -22,7 +22,8 @@ import {fromJS, List, Map, Set} from 'immutable';
   selector: 'sb-multiple-dropdown',
   templateUrl: './multiple-dropdown.component.html',
   styleUrls: ['./multiple-dropdown.component.scss'],
-  changeDetection: ChangeDetectionStrategy.OnPush
+  changeDetection: ChangeDetectionStrategy.OnPush,
```

Comment:
this is needed?

Reply:
I have just changed file from css to scss and updated in ts file. it will not impact anywhere

Reply:
`ViewEncapsulation` will not impact?

Reply:
I removed repeated css from some file and add in one component, so the css will be in multiple-dropdown component, so added here ViewEncapsulation so it will inherit in all components..Tested in collection-editor working fine. It won't impact.